torch/onnx/symbolic_opset9.py

```diff
@@ -355,16 +355,36 @@ def add(g, self, other, alpha=None):
             "Add", 9, 11, "Add between list of tensors not supported"
         )
 
-    # default alpha arg is to allow no-alpha add (aten add st overload no alpha)
-    if alpha and symbolic_helper._scalar(symbolic_helper._maybe_get_scalar(alpha)) != 1:
-        return symbolic_helper._unimplemented("add", "alpha != 1")
+    if alpha:
+        is_scalar_one = (
+            symbolic_helper._scalar(symbolic_helper._maybe_get_scalar(alpha)) == 1
+        )
+        is_value = symbolic_helper._is_value(alpha)
+        if is_value and not is_scalar_one:
+            # If alpha=1, we don't need to add "Mul" because "Mul(x, 1) = x".
+            other = g.op("Mul", other, alpha)
+        elif not is_value and not is_scalar_one:
+            return symbolic_helper._unimplemented(
```

Comment:
Looks like constant alpha != 1 should be easily supported. If you don't mind, could you please validate if this could be simplified down to
```python
if alpha and symbolic_helper._scalar(symbolic_helper._maybe_get_scalar(alpha)) != 1:
    other = g.op("Mul", other, alpha)
```

Reply:
It works and the last commit contains the corresponding change.